adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/impl/IMessageMapping.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.mqtt.impl;
+
+import org.eclipse.hono.adapter.mqtt.MqttContext;
+import org.eclipse.hono.util.ResourceIdentifier;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.mqtt.messages.MqttPublishMessage;
+
+/**
+ * A message mapping interface. This component requests mapping from another server is configured properly. The
```

Comment:
The first sentence is supposed to tell the reader a little about the nature of this class/interface. IMHO you should simply remove the first sentence.
Here is a nice [introduction to how to write good JavaDoc comments](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) ...